recipes/go-licenses/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set name = "go-licenses" %}
+{% set version = "0.0.35" %}
+{% set sha = "f29a4c695c3d0bedb950061032f97e90aa28d846" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/google/{{ name }}/archive/{{ sha }}.zip
+  sha256: fbd30c6704a02fdce0fbc1ebf8b07871a9c4db9fc87c14a5b08dfcdc1bc7739f
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('go') }}
+    - posix  # [win]
+
+test:
+  commands:
+    - which go-licenses  # [not win]
+    - where go-licenses  # [win]
```

Comment:
It is better to use existence tests: https://conda-forge.org/docs/maintainer/adding_pkgs.html#simple-existence-tests